Modules/Journey/QuestieSearchResults.lua

```diff
@@ -90,7 +90,8 @@ local function CreateShowHideButton(id)
 end
 
 function QuestieSearchResults:QuestDetailsFrame(details, id)
-    local name, requiredLevel, requiredRaces, objectivesText, startedBy, finishedBy = unpack(QuestieDB.QueryQuest(id, "name", "requiredLevel", "requiredRaces", "objectivesText", "startedBy", "finishedBy"))
+    local ret = QuestieDB.QueryQuest(id, "name", "requiredLevel", "requiredRaces", "objectivesText", "startedBy", "finishedBy") or {}
+    local name, requiredLevel, requiredRaces, objectivesText, startedBy, finishedBy = ret[1], ret[2], ret[3], ret[4], ret[5], ret[6]
```

Comment:
Is this pattern faster than the multiple single queries? If yes why not using it at the other places as well?

Reply:
For querying two fields QueryQuestSingle() is faster. I haven't tested 3+ fields.

Generally this PR intent is not speed up, but getting rid of unpack, which leads to bugs if there is `nil` within return values.